packages/dd-trace/src/config.js

```diff
@@ -545,8 +549,8 @@ class Config {
     this._setValue(defaults, 'traceId128BitGenerationEnabled', true)
     this._setValue(defaults, 'traceId128BitLoggingEnabled', false)
     this._setValue(defaults, 'tracePropagationExtractFirst', false)
-    this._setValue(defaults, 'tracePropagationStyle.inject', ['datadog', 'tracecontext'])
-    this._setValue(defaults, 'tracePropagationStyle.extract', ['datadog', 'tracecontext'])
+    this._setValue(defaults, 'tracePropagationStyle.inject', ['datadog', 'tracecontext', 'baggage'])
```

Comment:
A baggage is not a propagation style. If anything, the 2 baggage formats actually map 1:1 with the existing propagation styles.

Reply:
It's worth mentioning that OTEL considers baggage as a [propagator](https://opentelemetry.io/docs/specs/otel/context/api-propagators/)

Reply:
That design is flawed though. For example, they mention in the same doc that they will add a binary propagator in the future in addition to TextMap. But then they also refer to the data that is added to the carrier within a propagator as also a propagator. So how would you differentiate between baggage propagated by TextMap versus by Binary at that point?